code/videoDrawer.py

```diff
@@ -216,35 +216,31 @@ def frameOf(p, offset):
 lastTimestamp = float(lastParts[0])
 FRAME_COUNT = timestepToFrames(lastTimestamp+1)
 phonemeTimeline = []
-for i in range(len(schedules[4])):
-    parts = schedules[4][i].split(",")
-    timestamp = float(parts[0])
-    framestamp = timestepToFrames(timestamp)
+for elem in schedules[4]:
+    timestamp, _, phoneme = elem.split(",")
+    framestamp = timestepToFrames(float(timestamp))
     if i >= 1 and framestamp <= phonemeTimeline[-1][1]: # we have a 0-frame phoneme! Try to fix it.
```

Comment:
The index `i` is used here but you removed it

Reply:
Oh dear, you're right